src/models/flinkStatement.ts

```diff
@@ -0,0 +1,55 @@
+import { ThemeIcon, TreeItem, TreeItemCollapsibleState } from "vscode";
+import { ConnectionType } from "../clients/sidecar";
+import { IconNames } from "../constants";
+import { ConnectionId, EnvironmentId, IResourceBase, ISearchable } from "./resource";
+
+export class FlinkStatement implements IResourceBase, ISearchable {
+  connectionId!: ConnectionId;
+  connectionType!: ConnectionType;
+  iconName: IconNames = IconNames.FLINK_STATEMENT;
+
+  environmentId!: EnvironmentId;
+  computePoolId!: string;
+
+  name!: string;
```

Comment:
Also suggest growing an `id` getter property, returning `name` value. Name is guaranteed to be unique within the ccloud env.

<img width="806" alt="image" src="https://github.com/user-attachments/assets/274e3e46-8d60-455d-af42-aaf33c2db7e3" />

Reply:
Sounds good - done in https://github.com/confluentinc/vscode/pull/1399/commits/94b5da34738b4c12a6514ea1a46d5c140fe43e10